public/lib/commands/stripe_checkout/complete.liquid

```diff
@@ -1,16 +1,18 @@
 {% liquid
-  function object = 'modules/payments_stripe/commands/checkout_session/complete/build', object: object
-  function object = 'modules/payments_stripe/commands/checkout_session/complete/check', object: object
+  function object = 'modules/payments_stripe/commands/stripe_checkout/complete/build', object: object
+  function object = 'modules/payments_stripe/commands/stripe_checkout/complete/check', object: object
   unless object.valid
     return object
   endunless
 
   function object = 'modules/payments/commands/gateway_requests/receive', object: object.data.object, name: object.type, external_id: object.data.object.client_reference_id, request_url: request_url
   if object.valid
-    function object = 'modules/payments_stripe/commands/checkout_session/complete/map_request', object: object
+    function object = 'modules/payments_stripe/commands/stripe_checkout/complete/map_request', object: object
     function transaction = 'modules/payments/queries/transactions/find', id: object.external_id
     assign input = null | hash_merge: payment_status: object.status
     function object = 'modules/payments/commands/transactions/update_status', object: input, transaction: transaction, requester_id: requester.id
+  else
+    log object, type: "ERROR"
```

Comment:
```suggestion
    log object, type: "ERROR: modules/payments-stripe/commands/stripe_checkout/complete"
```

Reply:
We have a file name from where `log` was triggered in logs itself so I stopped to add a path to a file in type